sample-apps/radiology/lib/transforms/transforms.py

```diff
@@ -505,3 +506,39 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
             if d.get(cache_key) is None:
                 d[cache_key] = copy.deepcopy(d[key])
         return d
+
+
+class OrientationGuidanceMultipleLabelDeepEditd(Transform):
```

Comment:
Hi @diazandr3s , is there any test case to prove that the oriented coordinates are consistent with oriented volume?

Reply:
Hi @yiheng-wang-nv,

I don't have samples for this. But it makes sense to ensure the same orientation is applied to the clicks :)
I also asked @VishalJ99 to check if this works: https://github.com/Project-MONAI/MONAILabel/issues/1407#issuecomment-1697914123